File: cpp/bc1113.cpp
#include <iostream>

using namespace std;

int main()
{
	int i = 0, x, y;
	string msg = "";
	
	while(true)
	{
		cin >> x >> y;
		
		if(x == y) break;
		else if(x > y) msg = "Decrescente";
		else msg = "Crescente";
		
		cout << msg << endl;
	}
}

File: python/bc1113.py
while True:
    msg = ""
    vec = input().split()

    fVal = int(vec[0])
    sVal = int(vec[1])

    if fVal == sVal: break
    elif fVal > sVal: msg = "Decrescente"
    else: msg = "Crescente"

    print(msg)